crates/client/src/commander.rs

```diff
@@ -508,7 +508,7 @@ impl Commander {
     #[throws]
     pub async fn parse_program_client_imports(&self) -> Vec<syn::ItemUse> {
         let output = Command::new("cargo")
-            .arg("+nightly")
+            .arg("+nightly-2023-12-28-x86_64-unknown-linux-gnu")
```

Comment:
Why do we need a specific nightly version? It works for me If I use only +nightly

Reply:
If you update the nightly with `rustup update nightly` to v1.78, than there is an issue of ahash crate compilation. So forcing the nightly v1.77 is a workaround here. Changing the ahash version is not possible because it is pinned by some solana dependencies.